packages/sprotty/src/base/model/smodel-utils.ts

```diff
@@ -165,3 +165,14 @@ export function containsSome(root: SModelRoot, element: SChildElement): boolean
     const find = (elements: readonly SChildElement[]): boolean => elements.some(el => test(el) || find(el.children));
     return find([element]);
 }
+
+/**
+ * Transforms the local bounds all the way up to the last parent.
+ */
+export function transformToParentBounds(parent: SParentElement, bounds: Bounds) {
```

Comment:
Should it be named  `transformToRootBounds` because it considers all parents up to the root, not just the direct parent?

Reply:
The function has now been renamed to `transformToRootBounds`, as you suggested.